src/Compiler/FSComp.txt

```diff
@@ -1745,4 +1745,6 @@ featureReuseSameFieldsInStructUnions,"Share underlying fields in a [<Struct>] di
 3862,parsStaticMemberImcompleteSyntax,"Incomplete declaration of a static construct. Use 'static let','static do','static member' or 'static val' for declaration."
 3863,parsExpectingField,"Expecting record field"
 3864,tooManyMethodsInDotNetTypeWritingAssembly,"The type '%s' has too many methods. Found: '%d', maximum: '%d'"
-3865,parsOnlySimplePatternsAreAllowedInConstructors,"Only simple patterns are allowed in primary constructors"
\ No newline at end of file
+3865,parsOnlySimplePatternsAreAllowedInConstructors,"Only simple patterns are allowed in primary constructors"
+3866,tcParameterizedActivePatternArgumentCountNotMatch,"This active pattern requires %d parameter(s) and returns %d value(s), but the usage here matches %d argument(s) and %d return value(s)."
+3867,tcNoParameterActivePatternArgumentCountNotMatch,"This active pattern returns %d value(s), but the usage here matches %d return value(s)."
```

Comment:
These error numbers were used in #17055. Be sure to use new numbers when resolving the merge conflict.